guides/common/modules/proc_adding-custom-rpm-repositories-for-alma-linux-9.adoc

```diff
@@ -0,0 +1,33 @@
+[id="Adding_Custom_RPM_Repositories_for_Alma_Linux_9_{context}"]
+= Adding {customrpmtitle} Repositories for Alma Linux 9
+
+This example creates a product and repositories for Alma Linux 9.
+
+.Prerequisite
+* Download and import the http://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux-9[GPG public key] for yum repositories for Alma Linux 9 into {Project}.
+For more information, see xref:Importing_a_Custom_GPG_Key_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content > Products*.
+. Click *Create Product* to create a product named `Alma Linux 9`.
+For more information, see xref:Creating_a_Custom_Product_{context}[].
+. On the *Repositories* tab, click *New Repository* to create three repositories of type `yum` as follows:
++
+* *Alma Linux 9 BaseOS*
+** *Upstream URL*: `http://repo.almalinux.org/almalinux/9/BaseOS/x86_64/os/`
+* *Alma Linux 9 AppStream*
+** *Upstream URL*: `http://repo.almalinux.org/almalinux/9/AppStream/x86_64/os/`
+* *Alma Linux 9 extras*
+** *Upstream URL*: `http://repo.almalinux.org/almalinux/9/extras/x86_64/os/`
++
+For more information, see xref:Adding_Custom_RPM_Repositories_{context}[].
+. Click *Create Product* to create a product named `Alma Linux 9 client`.
```

Comment:
Should this be `Foreman EL9 Client` perhaps? It is the same for all EL9 flavors and if you're a shop with (for example) both RHEL and CentOS Stream, you can reuse the product, right?